docs/admin-guide/admin-configuration.md

```diff
@@ -194,6 +194,8 @@ Most modern filesystems buffer and flush RAM data to disk after a delay. The pur
 
 Deduplication is a storage space saving feature wherein only a single copy of specific content is maintained on disk while many different image manifests may hold references to that same content. The deduplication option (`dedupe`) is also available for supported cloud storage backends.
 
+Everytime zot starts running it will enforce the `dedupe` status on the existing storage, so changing from disabled to enabled and restarting zot will deduplicate all blobs found in storage, similarly changing from enabled to disabled will restore blobs to their original state in case of cloud storage, in case of filesystem storage there is no need to restore because hard links are used.
```

Comment:
Suggested rewording of this paragraph in both files:

"Upon startup, zot enforces the `dedupe` status on the existing storage. If the `dedupe` status upon startup is `true`, zot deduplicates all blobs found in storage, both local and remote.  If the status upon startup is `false`, zot restores cloud storage blobs to their original state. There is no need for zot to restore local filesystem storage if hard links are used."

Reply:
fixed, thank you!